jobs/stack_os_matrix.groovy

```diff
@@ -11,16 +11,15 @@ p.pipeline().with {
     stringParam('REFS', null, 'Whitespace delimited list of "refs" to attempt to build.  Priority is highest -> lowest from left to right.  "master" or branch from repos.yaml is implicitly appended to the right side of the list, if not specified; do not specify "master" explicitly.')
     stringParam('PRODUCTS', scipipe.canonical.products,
       'Whitespace delimited list of EUPS products to build.')
+    booleanParam('BUILD_DOCS', true, 'Build pipelines.lsst.io site')
+    booleanParam('PUBLISH_DOCS', true, 'Publish pipelines.lsst.io/v edition')
```

Comment:
How does it know where to publish that in the `/v` tree? Does it combine all the ticket branches that have been specified for the build?  I see some code later that looks like it might be doing that and replacing space with `-`.

Do we clean them out periodically?

Reply:
It's based on the git refs, yes, and it combines all ticket branches, as requested in the ticket.

I don't think we clear them out at present.  @jonathansick ?

Reply:
We don't delete old editions at present but that's a feature we have in mind for LSST the Docs.